docs/BR.md

```diff
@@ -1099,8 +1103,40 @@ No stipulation.
 
 ### 4.3.1 CA actions during certificate issuance
 
+#### 4.3.1.1 Manual authorization of certificate issuance for Root CAs
+
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Precertificate [RFC 6962] is issued, effective 2024-09-15 the CA SHOULD implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements. Effective 2025-03-15, the CA SHALL implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements.
+
+Methods used to produce a certificate containing the to-be-signed Certificate content include, but are not limited to:
+
+1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is not certified by a Certificate that chains to a publicly-trusted CA Certificate; or
+2. Specify a static value for the `signature` field of the Certificate ASN.1 SEQUENCE.
+
+CAs MAY implement their own certificate Linting tools, but CAs SHOULD use the Linting tools that have been widely adopted by the industry (see https://cabforum.org/resources/tools/). 
+
+CAs are encouraged to contribute to open-source linting projects, such as by:
+
+- creating new or improving existing lints,
+- reporting Requirements that are not covered by existing lints ("unlinted issues"), 
+- updating documentation of existing lints, and 
+- generating test certificates for positive/negative tests of specific lints.
+
+#### 4.3.1.3 Linting of issued Certificates
```

Comment:
I think this should be a sub-section of 4.3.1.2, as it is not an action taken by the CA prior to Certificate issuance.

Reply:
I thought about this when deciding on a separate number. The title of 4.3.1.2 is "Linting of to-be-signed Certificate content" which doesn't make sense to include suggestions/recommendations/requirements for linting of final certificates. That's why a separate same-level subsection was created. Similarly, the update section of linters applies to both "pre-sign" and "post-sign" linting.

Does that make sense?

Reply:
It would be clearer if we moved this section to 8.7.1. That would make it clear that this activity a part of periodic self-audits and not an activity performed before/during Certificate issuance.

Reply:
Thanks. I believe it's addressed in 27680a69fb505ad03790d4160fed9b391a7e2093. Please confirm.

Reply:
Moving to 8.7 itself is fine, although it will cause a divergence in the content of that section with other documents. Likely not material now, but might be when moving to BR of BRs.

Reply:
The BR of BRs will provide an opportunity for WGs to evaluate good practices of other WGs. My hope is that Linting will be considered a good practice in S/MIME and Code Signing WGs as well.